validation.scss

```diff
@@ -12,10 +12,9 @@
 	display: block;
 }
 
-@mixin vui-validation-bubble-content() {
+@mixin vui-validation-bubble-content-base() {
```

Comment:
A convention we've been following for this is to prefix the mixin with an `_`, so this would be come `_vui-validation-bubble-content`.